drools-osgi/drools-karaf-itests/src/it/drools-karaf-itests-kjar/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.drools</groupId>
+    <artifactId>drools-osgi</artifactId>
+    <version>7.0.0-SNAPSHOT</version>
```

Comment:
We will need to use property here, e.g. `${version.org.kie}`. Otherwise the version won't get changed automatically by Maven versions plugin.